examples/keystatic/app/keystatic/[[...params]]/page.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Page() {
+  return null
```

Comment:
Is this intentional?

Reply:
Hey 👋

Super late seeing this comment, sorry 😅

The page returning null is intentional because the app is rendered in the layout to avoid getting re-mounted on each route navigation.